include/graphblas/hyperdags/io.hpp

```diff
@@ -332,6 +332,38 @@ namespace grb {
 		return ret;
 	}
 
+	template<
```

Comment:
Perf semantics are missing (just a comment that it inherits the performance semantics of the underlying backend would be sufficient)